core/controllers/suggestion.py

```diff
@@ -342,13 +350,49 @@ def get(self, target_type, suggestion_type):
             target_type: str. The type of the suggestion target.
             suggestion_type: str. The type of the suggestion.
         """
+        topic_name = self.request.get('topic_name', None)
```

Comment:
Let's move this below line 355 where we do some validation.

Reply:
Done